.rubocop.yml

```diff
@@ -100,6 +100,10 @@ RSpec/DescribeClass:
 RSpec/VerifiedDoubleReference:
   Enabled: false
 
+RSpec/MultipleMemoizedHelpers:
+  Exclude:
```

Comment:
How about just setting a reasonable Max value instead of excluding specific files?

Reply:
I set this to 15 in my recent PR

Reply:
cool, I'll rebase and see if that helps

Reply:
seems like that did it.